plugins/chat/assets/javascripts/discourse/components/chat-message-actions-desktop.js

```diff
@@ -54,6 +54,10 @@ export default class ChatMessageActionsDesktop extends Component {
         this.context
       );
 
+      if (!messageContainer) {
+        return;
+      }
+
```

Comment:
A small bug which happening in styleguide when you don't have the actions menu in the component, as it's part of the channel